src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage/index.unit.spec.jsx

```diff
@@ -5,16 +5,56 @@ import moment from 'moment';
 import {
   createTestStore,
   renderWithStoreAndRouter,
+  setCommunityCareFlow,
 } from '../../../mocks/setup';
 import userEvent from '@testing-library/user-event';
 
 import DateTimeRequestPage from '../../../../new-appointment/components/DateTimeRequestPage';
 import { FETCH_STATUS } from '../../../../utils/constants';
 import { waitFor, within } from '@testing-library/dom';
 import { Route } from 'react-router-dom';
+import { mockFetch } from 'platform/testing/unit/helpers';
+
+// Chooses the first valid morning request slot available, moving
+// to the next month if necessary
+async function chooseMorningRequestSlot(screen) {
```

Comment:
Another helper function, since picking a valid request slot is kinda complicated